system/jlib/jlog.cpp

```diff
@@ -2645,6 +2647,7 @@ MODULE_EXIT()
     thePassNoneFilter = nullptr;
     thePassLocalFilter = nullptr;
     thePassAllFilter = nullptr;
+    ::Release(logAccessor);
```

Comment:
also set logAccessor to null - just to avoid problems if this is called twice.  (Unlikely but we've been chasing similar bugs elsewhere.)